ui/apps/platform/cypress/integration/vulnmanagement/entitypages.test.js

```diff
@@ -330,7 +330,6 @@ describe('Entities single views', () => {
             cy.get(`${selectors.tableRows}`).eq(1).click();
         }, entitiesKey);
 
-        cy.get('button:contains("Fixable CVEs")').click();
```

Comment:
This is related to removing policies from the deployments page. Will need to go back and re check tests related to policies in vulnerability management, just wanted to get tests back to a passing state with this merge.